app/interactors/tags/update_interactor.rb

```diff
@@ -29,9 +28,10 @@ def find_and_lock_edition
   end
 
   def update_revision
-    context.revision_updater = Versioning::RevisionUpdater.new(edition.revision, user)
-    revision_updater.assign(tags: update_params)
-    context.revision = revision_updater.next_revision
+    updater = Versioning::RevisionUpdater.new(edition.revision, user)
+    updater.assign(tags: update_params)
+    context.fail! unless updater.changed?
```

Comment:
This is basically equivalent to `throw "Foo" if bar`; in these cases our linter usually enforces a newline before the next instruction in the block.

Should we do the same for `context.fail!` (even though the linter doesn't recognise it as an exception)? I notice it had a newline prior to this change.
Better yet, is there a way we can get the linter to recognise that these instructions are exception-raisers?

Reply:
You've spotted my compulsion to remove redundant newlines 😄. While I do agree with using linting to keep us consistent, I'd like to suggest we pass on it this time, and we can revisit if it becomes a recurring comment. Is that OK?

Reply:
Sure 👍